march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -41,7 +41,7 @@ int main(int argc, char** argv)
     return 1;
   }
 
-  const double loop_hz = ros::param::param("~loop_hz", 100.0);
+  const double loop_hz = ros::param::param("~loop_hz", 250.0);
```

Comment:
Good idea, could it be useful to add a parameter that specifies a multiplier of the frequency, so that we can set the controller frequency lower, but still a multiple of the ethercat cycle?

Reply:
Yes, I think when you read the ecatcycletime in the builder that we can make a parameter out of it. We should make the frequency here dependent on that parameter.